DorisProject/src/app/pages/modal/modal.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, Inject, OnInit} from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Replace `Component,·Inject,·OnInit` with `·Component,·Inject,·OnInit·`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9269670)
```suggestion
import { Component, Inject, OnInit } from '@angular/core';
```